test/androidTest/java/org/thoughtcrime/securesms/espresso/BaseRecyclerHelper.java

```diff
@@ -0,0 +1,66 @@
+package org.thoughtcrime.securesms.espresso;
+
+import android.support.test.espresso.AmbiguousViewMatcherException;
+import android.support.test.espresso.NoMatchingViewException;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.RecyclerViewActions.scrollToPosition;
+import static android.support.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.thoughtcrime.securesms.espresso.ViewMatchers.atPosition;
+
+class BaseRecyclerHelper<T> extends Helper {
+
+    /* ASSERTIONS */
+
+    public T assertIdAt(int id, int position, boolean checkVisible) {
```

Comment:
I want to split id and visible assertions.

Reply:
Oh ok cool I wasn't sure if you were finished the tests or not

Reply:
they are functionally finished, but I wanna make em nicer. was getting frustrated with espresso at the end of it ...